tests/conftest.py

```diff
@@ -21,6 +24,36 @@ def docker():
     return libdocker.Client(version='auto')
 
 
+@pytest.yield_fixture(scope='session')
+def ssl_folder(docker_ip_address):
```

Comment:
maybe we can generate and store (commit) ssl cert/key in tests/data for example... and use it in tests